docs/support/faq/pouta-openstack-rescue-mode.md

```diff
@@ -44,27 +42,50 @@ Check your instance Console Log (web UI: Instances -> <your instance> -> Log)
 
 The log says that the instance couldn't boot because it can't find root "Kernel panic - not syncing: VFS: Unable to mount root fs onunknown-block(0,0)". The fix is to use (some) previous, working kernel. Since you can't boot the server, you have to make the fix to the Volume (boot files) by using another instance.
 
-## How to fix the issue
-
 Note, that you will modify grub, there are probably better ways to do it, so do not be afraid to see what other guides there are on the internet. This FAQ is mostly meant to show that there is a command named `nova rescue`. There is also a command named `openstack server rescue` which is almost the same as `nova rescue` but is missing the `--image` flag which is almost *always* required when rescuing servers.
 
 1. shutdown the instance <pre><code>openstack server stop $INSTANCE_UUID</pre></code>
 2. Use this command to rescue the instances. <pre><code>nova rescue $INSTANCE_UUID --image CentOS-7</pre></code>
-1. Make sure that the instance is in rescue mode with <pre><code>openstack server show $INSTANCE_UUID</pre></code>
-3. ssh into the instances.
-3. Check what volumes you have. If you don't have any other volumes attached it should looks something like this:
+
+In this step, in case you have many images with the same name, you need to use the image ID instead of the image name (CentOS-7). In order to find the ID of the image use the following command: <pre><code> openstack image list </pre></code>.
+
+Also, if your intance is an Ubuntu instead of CentOS, then select Ubuntu image ID instead. 
+
+3. Make sure that the instance is in rescue mode with <pre><code>openstack server show $INSTANCE_UUID</pre></code>
+
+4. ssh into the instances.
+
+In case that you have this message in the terminal:  <pre><code>WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!</pre></code>
+Fix it by removing the line of your instance IP address from the file ~/.ssh/known_hosts
+
+An alternative way is the execution of the following command:<pre><code>ssh-keygen -f "/root/.ssh/known_hosts" -R "$INSTANCE_IP"</pre></code>
```

Comment:
That probably won't work as generally people shouldnt really be sshing from root.  ~/.ssh/known_hosts makes more sense

Reply:
Ok. I will update it.

Reply:
Is this fixed?